_data/projects/mediawiki.yml

```diff
@@ -0,0 +1,12 @@
+name: MediaWiki
+desc: The collaborative editing software that runs Wikipedia
+site: https://www.mediawiki.org
+tags:
+- mediawiki
+- wiki
+- web
+- cms
+- php
+upforgrabs:
+  name: Easy
+  link: https://phabricator.wikimedia.org/tag/easy/
```

Comment:
I thought about that (I didn't like linking to a landing page either), but I'm not sure how permanent that link is. Should I change?